src/cpacs_other/CPACSFarField.cpp

```diff
@@ -27,44 +26,16 @@ namespace tigl
 {
 namespace generated
```

Comment:
Why is CPACSFarField still in the `generated` namespace? This can be misleading.

Reply:
To make clear, that this file was generated (once). In case we are regenerating it automatically.

Otherwise, I can also merge generated::CPACSFarField and CCPACSFarField into one class.